pkg/server/start.go

```diff
@@ -118,6 +123,8 @@ func NewCommandStartAdapterServer(stopCh <-chan struct{}) *cobra.Command {
 		"disregard failing to create collectors for incompatible HPAs")
 	flags.DurationVar(&o.MetricsTTL, "metrics-ttl", 15*time.Minute, "TTL for metrics that are stored in in-memory cache.")
 	flags.DurationVar(&o.GCInterval, "garbage-collector-interval", 10*time.Minute, "Interval to clean up metrics that are stored in in-memory cache.")
+	flags.BoolVar(&o.ScalingScheduleMetrics, "scaling-schedule", o.ScalingScheduleMetrics, ""+
+		"whether to enable time-based ScalingSchedule metrics")
```

Comment:
Alternatively we could enable it only if the CRD is available. I think this is fine for now, but there may be users who doesn't want to use e.g. `ClusterScalingSchedule` and they could control it by not deploying the CRD for instance.

Just an idea, doesn't have to be done now at all.

Reply:
It's a good one, I will add a note to do that in another PR.

Reply:
#316 so I don't forget.